lips/lip-8.md

```diff
@@ -17,14 +17,25 @@ An extension of the Off-Chain protocol to provide support for more advanced merc
 
 Version 0 of the Off-Chain Protocol is described in [LIP 1](https://lip.libra.org/lip-1/).  Version 1 as described here is an extension of the Off-Chain Protocol and adds features to support more advanced functionality - particularly targeted to support merchant use-cases.  This is inclusive of pull payments, recurring payments, and auth/capture flows.
 
+---
+# Disclaimer
+---
+
+This LIP does not contain the initial phase of a sub-account discovery that is required to start negotiating merchant scenarios.
+The process described below starts at the phase that both sides (particularly the biller side) have the relevant subaddresses. 
+For the discovery phase, there are currently two leading methods:
```

Comment:
I am not sure we want to include any statements here on discovery methods, since this is an evolving area and there is currently no method(s) that Libra has standardized. I would rephrase and say that the industry has mechanisms such as [...] and others.